### ISSUE TITLE
Parameterize the display and size

### ISSUE THREAD

**Issue by MadOtis** [TomWhitwell/SlowMovie#5]
If you parameterize the EPD board and display size (for example, in my case I'm using an 880x528 sized 7x5 HD epd), it would be easier to simply specify a board/size instead of rooting through code to find where the EPD is defined and used.  Granted, this does not work on 'b' variants because of differences in API, but this can also be filtered and alerted upon if someone tries to use a multi-color display.

**Comment by missionfloyd** [comment]
I've actually simplified this in my fork. All that's needed to specify a display is to change the [import](https://github.com/missionfloyd/SlowMovie/blob/cd6c2dafc4235462d9efb843f0f59b00158a5955/slowmovie.py#L19).

**Comment by NickAnsy** [comment]
Forgive me in advanced, I'm a total noob and just learning how to code. I have the 12.48in screen. I am having trouble getting it to work. All of the demos in the screens folder work (12.48in.py and Weather.py). It's telling me it can't find the waveshare module for the screen. I figured you tackled this same issue because of the different screen size. Thanks!

**Comment by missionfloyd** [comment]
@NickAnsy I don't have the 12.48" screen to test, but try this.
https://github.com/missionfloyd/SlowMovie/tree/12.48inch

Copy the lib folder from the waveshare 12.48" repo to the same folder as slowmovie.py

**Comment by Estacaco** [comment]
I'm having similar issues but with the 2.13" v2 screen (yes I know it's tiny but its what I have on hand to play/test with) I've changed the script to reflect my 2.13" screen and pixel parameters but keep getting errors when trying to run the script. I had to change the init and clear to match what is being used in the example script to get it to even flash/clear the screen, but still has other issues with the script.

**Comment by TheQwerty** [comment]
@Estacaco That screenshot seems to show it being unable to find the video file. Are you certain it exists at `/home/pi/SlowMovie/Videos/test.mp4` ?

**Comment by Estacaco** [comment]
it sure is, I tried missionfloyds fork and i get the same issue after repairing some things (my screen requires init(epd.FULL_UPDATE) and clear(0xFF) or the script errors)

**Comment by Estacaco** [comment]
The only thing that I can think of is different calls/commands with the different drivers.

**Comment by Estacaco** [comment]
So if i give it an arbitrary number for the number of frames instead of having it count them (eg:1800) it gets by the line I posted previously but then continues to fail more things having to do with ffmpeg and grabbing the jpeg.

`pi@raspberrypi:~/SlowMovie2 $ sudo python helloworld.py
/home/pi/SlowMovie2/Videos/test.mp4
Traceback (most recent call last):
  File "helloworld.py", line 66, in <module>
    generate_frame(inputVid, 'grab.jpg', msTimecode, width, height)
  File "helloworld.py", line 25, in generate_frame
    .run(capture_stdout=True, capture_stderr=True)
  File "/usr/local/lib/python2.7/dist-packages/ffmpeg/_run.py", line 320, in run
    overwrite_output=overwrite_output,
  File "/usr/local/lib/python2.7/dist-packages/ffmpeg/_run.py", line 285, in run_async
    args, stdin=stdin_stream, stdout=stdout_stream, stderr=stderr_stream
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory`

**Comment by Estacaco** [comment]
So I was able to resolve it wiht reinstalling ffmpeg. now im just having issues with it making and grabbing the jpeg with pil lol

/home/pi/SlowMovie/Videos/test.mp4
Traceback (most recent call last):
  File "helloworld.py", line 69, in <module>
    pil_im = Image.open("grab.jpg")
  File "/usr/local/lib/python2.7/dist-packages/PIL/Image.py", line 2766, in open
    fp = builtins.open(filename, "rb")
IOError: [Errno 2] No such file or directory: 'grab.jpg'

**Comment by TomWhitwell** [comment]
Sorry you’re having so many problems - did you use Noobs, or build your own SD Card? I

**Comment by Estacaco** [comment]
No worries Tom thanks for doing all the hard leg work! I actually got it working last night. After I reinstalled FFMPEG I didnt reboot, so that seemed to fix the remaining issues! The only note to make would be some of the waveshare screens don't work with blank init() and clear() and require init(epd.FULL_UPDATE) clear(0xFF) to make it work.
![20201111_221300](https://user-images.githubusercontent.com/44181620/98957534-e7710380-24c6-11eb-91be-2a58eaa439da.jpg)
![20201111_233620](https://user-images.githubusercontent.com/44181620/98957579-f0fa6b80-24c6-11eb-9385-d58359694f8f.jpg)
![20201112_083604](https://user-images.githubusercontent.com/44181620/98957616-f9eb3d00-24c6-11eb-8f8c-92451ed241a1.jpg)
